docs/source/about/guides/linux/headless_ssh.rst

```diff
@@ -0,0 +1,481 @@
+
+         sudo dnf install openssh-server
+
+Next make sure the OpenSSH daemon is enabled to run when the system starts.
+
+.. tab:: SystemD
+
+    .. code-block:: bash
+
+      sudo systemctl enable sshd.service
+      sudo systemctl start sshd.service  # Starts the service now
+      sudo systemctl status sshd.service  # See if the service is running
+
+.. tab:: Runit
+
+   .. code-block:: bash
+
+      sudo ln -s /etc/runit/sv/sshd /run/runit/service  # Enables the OpenSSH daemon to run when system starts
+      sudo sv start sshd  # Starts the service now
+      sudo sv status sshd  # See if the service is running
+
+.. tab:: OpenRC
+
+    .. code-block:: bash
+
+        rc-update add sshd  # Enables service
+        rc-status  # List services to verify sshd is enabled
+        rc-service sshd start  # Starts the service now
+
+**Disabling PAM in sshd**
+
+I noticed when the ssh session is disconnected for any reason, ``pulseaudio`` would disconnect.
+This is due to PAM handling sessions. When running ``dmesg``, I noticed ``elogind`` would say removed user session.
+In this `Gentoo Forums post <https://forums.gentoo.org/viewtopic-t-1090186-start-0.html>`__,
+someone had a similar issue. Starting the X server in the background and exiting out of the console would cause your
+session to be removed.
+
+.. caution::
+   According to this `article <https://devicetests.com/ssh-usepam-security-session-status>`__
+   disabling PAM increases security, but reduces certain functionality in terms of session handling.
+   *Do so at your own risk!*
+
+Edit the ``sshd_config`` file with the following to disable PAM.
+
+.. code-block:: text
+
+   usePAM no
+
+After making changes to the ``sshd_config``, restart the sshd service for changes to take effect.
+
+.. tip::
+   Run the command to check the ssh configuration prior to restarting the sshd service.
+
+   .. code-block:: bash
+
+      sudo sshd -t -f /etc/ssh/sshd_config
+
+   An incorrect configuration will prevent the sshd service from starting, which might mean
+   losing SSH access to the server.
+
+.. tab:: SystemD
+
+    .. code-block:: bash
+
+      sudo systemctl restart sshd.service
+
+.. tab:: Runit
+
+    .. code-block:: bash
+
+      sudo sv restart sshd
+
+.. tab:: OpenRC
+
+    .. code-block:: bash
+
+      sudo rc-service sshd restart
+
+
+Virtual Display Setup
+^^^^^^^^^^^^^^^^^^^^^
+
+As an alternative to a dummy dongle, you can use this config to create a virtual display.
+
+.. important::
+   This is only available for NVidia GPUs using Xorg.
+
+.. code-block:: xorg.conf
+
+   Section "ServerLayout"
+       Identifier "TwinLayout"
+       Screen 0 "metaScreen" 0 0
+   EndSection
+
+   Section "Monitor"
+       Identifier "Monitor0"
+       Option "Enable" "true"
+   EndSection
+
+   Section "Device"
+       Identifier "Card0"
+       Driver "nvidia"
+       VendorName "NVIDIA Corporation"
+       Option "MetaModes" "1920x1080"
+       Option "ConnectedMonitor" "DP-0"
```

Comment:
Made a note that the name of the mode and be found with running `xrandr`. The simplest option is to choose the active mode when the monitors  are connected. This will most likely be `HDMI-0` and `DP-0`